cypress/fixtures/mentors/get.json

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
If not allof the data necessary to run the test, whatdo you think of reducing its size (by not including all the tags or returning the minimum size of base64 avatar images to keep the repo's weight as small as possible.

Reply:
That works for me.  I will:

- trim the tags and countries down to only what the two mock mentors require.
- look for slimmer base64-encoded images (although I'll be sad to lose the kittens)